LibreNMS/Alert/Transport/Telegram.php

```diff
@@ -79,11 +84,18 @@ public static function configTemplate()
                     'name' => 'telegram-token',
                     'descr' => 'Telegram Token',
                     'type' => 'text',
+                ],
+                [
+                    'title' => 'Format',
+                    'name' => 'telegram-format',
+                    'descr' => 'Telegram format',
+                    'type' => 'text',
```

Comment:
Change this from text to select and add the options that telegram supports.

See `LibreNMS/Alert/Transport/Hue.php`